NSS-Day-Cohort-49/tabloid-mvc-tabloid-mvc-cassettes
# Pr initial check

#### Changes Made
1. Added ability to register a user. 
?
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout pr-initial-check
    ```
2. Open a new Terminal tab (?T) and navigate to the server directory.
3. Test app functionality.
    > navigate to the home page and make sure the current user you are using is logged out. 
    > click on either REGISTER button
    > fill out the required fields
    > make sure the new user is added to your database
    > login with that user and check all functionality that we have available up to this point
